docroot/profiles/custom/cgov_site/modules/custom/cgov_core/cgov_core.module

```diff
@@ -211,9 +211,12 @@ function __cgov_core_text_filter_modify($form_element) {
  * language for the UI pages to English.
  */
 function cgov_core_user_presave(EntityInterface $entity) {
-  if ($entity->isNew()) {
+  if ($entity->isNew() && ($entity->preferred_admin_langcode->value !== 'no-preference')) {
     $entity->set('preferred_admin_langcode', 'en');
   }
+  if ($entity->preferred_admin_langcode->value == 'no-preference') {
```

Comment:
You also need to add a comment here that this is support the drush command, and that we use `no-preference` because ALL new users on pre-save have a value for this field whether or not it is empty. So our command needs to be able to differentiate when we intentionally want it empty, vs when we what to have it empty so it defaults to english. (P.S. Make sure to put the command name in here).